### PR TITLE
Fixes for Conzoco

### PR DESCRIPTION
**Pull request by srevinsaju**
Needs _extensive_ testing; I have already done maximum testing I can. If there is some user who knows how to decipher the Sri Lankan map. it would be useful in testing the coverage

@chimosky

### CODE REVIEW

File: activity.py [hunk 1]
```diff
@@ -14,9 +22,47 @@ def __init__(self, handle):
...
+        toolbar_box.toolbar.insert(activity_button, 0)
+        activity_button.show()
+
+        description_item = DescriptionItem(self)
+        toolbar_box.toolbar.insert(description_item, -1)
+        description_item.show()
+
+        share_button = ShareButton(self)
+        toolbar_box.toolbar.insert(share_button, -1)
+        share_button.show()
```
Comment (JuiP) [inline activity.py]: This button is not required.
[ShareButton](https://github.com/sugarlabs/sugar-toolkit/blob/16e7854c139cd0c9c720a27754ad14ff3ea7a634/src/sugar/activity/widgets.py#L145) definition disables the button because max_participants = 1 in this activity

Comment (srevinsaju) [reply activity.py]: Thanks for noting that; I might have copied it from elsewhere;

Comment (srevinsaju) [reply activity.py]: Have addressed this in https://github.com/sugarlabs/iknowSriLanka/pull/5/commits/3d2a7bb1bd3de7b97ead571d9e30ace33e756dec

**Comment by srevinsaju** [comment]
@JuiP Please review now; I have fixed what [you have mentioned](https://github.com/sugarlabs/iknowSriLanka/pull/5#discussion_r422664694)

**Comment by JuiP** [comment]
Tested; works fine :)
I also noticed the stats show me incorrect Total time: (43.333 minutes in my case which is not possible)

**Comment by srevinsaju** [comment]
@JuiP I had noticed that, but I get very less numbers, for example 3.3333333333333333333333 or 4.3333333333333333333333; Need to add a float fix for it. I am not sure for 43.333 extra minutes; Did you have the same problem with [iknowIndia](https://github.com/sugarlabs/iknowIndia)? If yes, it might be a problem with each and every `conzoco` activity

**Comment by JuiP** [comment]
> Did you have the same problem with iknowIndia? If yes, it might be a problem with each and every conzoco activity

I have not tested other iknow activities, this was the first one so I'm not sure

File: conozco.py [hunk 2]
```diff
@@ -1047,7 +1047,7 @@ def load_stats(self):
                         val = f.readline()
                         val = val.strip('\n')
                         if not(val == ''):
-                            l[i] = int(val)
+                            l[i] = int(float(val))
```
Comment (chimosky) [inline conozco.py:1066]: Can you share the traceback this fixes? I don't see a reason why you'll convert to `float` then `int`.

Comment (srevinsaju) [reply conozco.py:1066]: @chimosky  In your comment https://github.com/sugarlabs/iknowSriLanka/pull/4#issuecomment-626194653 you had mentioned of an error

```bash
Cannot load stats invalid literal for int() with base 10: '0.0'
```

Small reproducer
```python
Python 3.8.2 (default, Apr  8 2020, 14:31:25) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> int('5.555')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: '5.555'
>>> float('5.5555')
5.5555
>>> int(float('5.5555'))
5
>>> 
```

So the real error of the previous error is 
```
ValueError: invalid literal for int() with base 10: '5.555'
```
This was the reason behind the fix

Comment (chimosky) [reply conozco.py:1066]: Yeah, thanks for that.

**Comment by chimosky** [comment]
@srevinsaju you said "Need to add a float fix for it.", can you fix that as I'm not able to test.

**Comment by srevinsaju** [comment]
@chimosky I have added the float fix on https://github.com/sugarlabs/iknowSriLanka/pull/5/commits/5fe59180d807f453fe0294c4628e7d77525780d2

Is there anything I can do more on this PR?

Seems like we need to unify all the `conzoco.py` in all `iknow*` activity

**Comment by chimosky** [comment]
> Is there anything I can do more on this PR?

Reviewed, thanks. The PR  is good.
Tested a little, don't know much about SriLanka.

> Seems like we need to unify all the conzoco.py in all iknow* activity

I agree with @quozl here https://github.com/sugarlabs/iknowIndia/pull/9#issuecomment-631811713.
